### PR TITLE
(recipe) add recipe for ghost cms

### PR DESCRIPTION
**Pull request by markhughes**
Closes: n/a

### What are the changes and their implications?

Adds ghost as a recipe! 

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [x] Documentation added/updated (https://github.com/blitz-js/blitzjs.com/pull/553)

### CODE REVIEW

**Comment by markhughes** [comment]
@flybayer a handful of tests have timed out, doesn't look like I can re-run them :(

File: recipes/ghost/index.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  paths,
+  RecipeBuilder,
+} from "@blitzjs/installer"
+import j from "jscodeshift"
+import {Collection} from "jscodeshift"
+import path from "path"
+
+export default RecipeBuilder()
+  .setName("Ghost")
+  .setDescription("Use Ghost in blitz.")
```
Comment (flybayer) [inline recipes/ghost/index.ts:14]: Is this change accurate?

```suggestion
  .setDescription("Access your Ghost CMS directly in blitz via the Ghost SDK.")
```

**Review by flybayer — COMMENTED** [review]
@markhughes I'm so sorry for the long delay on this!

Looks great. Can you open a PR to the docs to document addBlitzMiddleware?

File: recipes/ghost/index.ts (hunk 1 above)
Comment (markhughes) [reply recipes/ghost/index.ts]: Much better wording :)

**Comment by markhughes** [comment]
> @markhughes I'm so sorry for the long delay on this!

No problem, life happens! 

> Looks great. Can you open a PR to the docs to document addBlitzMiddleware?

Great idea. I'll do that :)

**Comment by markhughes** [comment]
Documentation added @ https://github.com/blitz-js/blitzjs.com/pull/553 :)

File: recipes/ghost/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@blitzjs/recipe-ghost",
+  "private": true,
+  "version": "0.40.0-canary.7",
```
Comment (flybayer) [inline recipes/ghost/package.json:4]:
```suggestion
  "version": "0.40.0-canary.9",
```

File: recipes/ghost/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "ghost"
+  ],
+  "author": "Mark Hughes <m@rkhugh.es>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/blitz-js/blitz/issues"
+  },
+  "homepage": "https://github.com/blitz-js/blitz#readme",
+  "dependencies": {
+    "@blitzjs/installer": "0.40.0-canary.7",
```
Comment (flybayer) [inline recipes/ghost/package.json:26]:
```suggestion
    "@blitzjs/installer": "0.40.0-canary.9",
```

**Pull request merged**